bin/release.sh

```diff
@@ -7,16 +7,37 @@ if [[ "$1" == "--dry-run" ]]; then
   DRY_RUN=true
   NPM_ARGS=" --dry-run"
   echo "Starting a dry run release..."
+else
+  DRY_RUN=false
+  echo "Starting a pre-release..."
```

Comment:
Should this be "release" instead of "pre-release"?

```suggestion
  echo "Starting a release..."
```

Reply:
I don't think so because its doing a pre-release before actually releasing. See line 35 is where it states at its now starting a release.

Reply:
Do you think it warrants a two-step process?

Reply:
Oh I missed that line below.

I'm not sure it needs to be two steps, I think the two things can go together?

Reply:
I've merged the two steps into a single step now.